src/Signer.php

```diff
@@ -75,7 +75,7 @@ private function generateSignature(string $value): string
      */
     private function splitSignatureFromSignedValue(string $signedValue): array
     {
-        $pos = strrpos($signedValue, '.');
+        $pos = strrpos($signedValue, ',');
```

Comment:
I'm not so sure about this.. On one hand it seems ok as a workaround, although I would like to still have BC here for `.` separated cookies so old cookies still work after deploying the new code (i.e. it should split using the last dot or comma found in the string if any).

On the other hand, it seems like the root cause is that you are signing session cookies, and https://github.com/nelmio/NelmioSecurityBundle/blob/7cc3a47ee6f57202ab53c5b1d0378e1a4e10d1a4/src/EventListener/SignedCookieListener.php#L57-L61 does not update $_COOKIE directly so the NativeSessionHandler does not see the unsigned-cookie value, and it actually uses the signed value as session id which to me sounds still borked.

So IMO we should probably write to $_COOKIE to unsign the cookies as well as writing to the request.. Any opinion here @nicolas-grekas from the Symfony/session perspective?

Reply:
Alternative fix would be to NOT sign the session cookie ever even if `*` is configured for signed cookies, because really that id should anyway be verified in the session storage to prevent session fixation, so I don't think signing this actually improves anything security wise.